### PR TITLE
improved useWatch test

### PR DESCRIPTION
**Pull request by keiya01**

### CODE REVIEW

File: src/useWatch.test.tsx [hunk 1]
```diff
@@ -5,11 +5,10 @@ import {
   fireEvent,
   act as actComponent,
 } from '@testing-library/react';
-import { renderHook, act } from '@testing-library/react-hooks';
+import { renderHook } from '@testing-library/react-hooks';
 import { useForm } from './useForm';
 import { useWatch } from './useWatch';
 import * as generateId from './logic/generateId';
-import { reconfigureControl } from './__mocks__/reconfigureControl';
```
Comment (bluebill1049) [inline src/useWatch.test.tsx:12]: bye!

File: src/useWatch.test.tsx [hunk 2]
```diff
@@ -276,42 +245,6 @@ describe('useWatch', () => {
...
-          },
-          name: 'test',
-          defaultValue: 'test',
-        }),
-      );
-
-      act(() => mockControl.watchFieldsHookRenderRef.current['0']());
-
-      expect(result.current).toEqual('test');
-    });
```
Comment (bluebill1049) [inline src/useWatch.test.tsx:314]: wow duplicated...

**Review by bluebill1049 — APPROVED** [review]
LGTM!

**Comment by keiya01** [comment]
@bluebill1049 Thank you for reviewing!

**Pull request merged**